### ISSUE TITLE
Vmq-admin session show: session not found with --topic

### ISSUE THREAD

**Issue by tinto82** [vernemq/vernemq#966]
Hello,
our production environment is made up of at least 3 nodes (max 5 nodes) running with Kubernetes on Google Cloud Platform. Over that is running an official docker image "erlio/docker-vernemq:1.5.0", in particular a cluster made up of 3 PODs.

If i get the sessions list with the command **vmq-admin session show --topic --is_online --user --limit=5000** the number of records shown is less than the number of records returned by the command **vmq-admin session show --is_online --user --limit=5000**. Why?

Thank you

**Comment by larshesel** [comment]
Hi, that's interesting.

How many records are returned in each case? Are there any errors/warnings in the logs?

It could be that some of the queries time out (this is printed in the log), you can increase the time the command waits for each query for each session to respond using `--rowtimeout=milliseconds`. The default is 100.

**Comment by tinto82** [comment]
Hi @larshesel 

at the moment for example the first command return 4056 record, the second one 4060. I launched the 2 commands at the same time.
This is our logs configuration:

```# log setting 
log.console = console
log.console.file = /vernemq/log/console.log
log.console.level = info
log.error.file = /dev/null

log.crash = on
log.crash.file = /etc/vernemeq/crash.log
log.crash.maximum_message_size = 64KB
log.crash.size = 10MB
log.crash.rotation = $D0
log.crash.rotation.keep = 5
```

but i can't find console.log, crash.log or error.log in Vernemq PODs, i can see only logs provided by Kubernetes and there i can't find warning or errors.
Thank you

**Comment by larshesel** [comment]
Could it be that there are actually clients disconnecting and connecting all the time? You could for instance sample `mqtt.socket_open - mqtt.socket_close` over time (every second or similar).

**Comment by tinto82** [comment]
No, is a device similar to the others. If i look at the result of second command the flag **is_online** is true.
I made a test with another device in the same situation: i used the command "vmq-admin session disconnect" and then the client made a new register+subscribe and now if i execute the first command i can see the record correctly.

**Comment by tinto82** [comment]
Today other 3 clients are in the same situation.
What can i do to provide you more information?

**Comment by tinto82** [comment]
Does our logs configuration look correct?

**Comment by larshesel** [comment]
> log.crash.file = /etc/vernemeq/crash.log

this looks wrong, firstly it's unusual to log anything to `/etc/`, secondly `vernemeq` looks like a typo, you probably meant `vernemq`.

You have too make sure that you see normal logs, for instance there should always be something logged in the `console.log` file when vernemq starts. If that happens then logging works.

In general all the log files should be created, even if  nothing is logged at all.

**Comment by larshesel** [comment]
Btw, just realized when using docker the console logs will only be logged to STDOUT and not the log files unless `DOCKER_VERNEMQ_LOG__CONSOLE` is set to `both` , see https://github.com/vernemq/docker-vernemq#logging
